python/src/Sample.i

```diff
@@ -58,20 +58,46 @@ Sample.__getattribute__ = Sample___getattribute__
 
 def Sample__repr_html_(self):
     """Get HTML representation."""
+    from openturns.common import ResourceMap
     html = '<TABLE>'
     desc = self.getDescription()
+    printEllipsisThreshold = ResourceMap.GetAsUnsignedInteger("Sample-PrintEllipsisThreshold")
+    printEllipsisSize = ResourceMap.GetAsUnsignedInteger("Sample-PrintEllipsisSize")
```

Comment:
you can probably do printEllipsisSize = openturns.common.ResourceMap.GetAsUnsignedInteger("Sample-PrintEllipsisSize")